labelbox/schema/dataset.py

```diff
@@ -410,9 +410,8 @@ def export_data_rows(self, timeout_seconds=120) -> Generator:
                 response.raise_for_status()
                 reader = ndjson.reader(StringIO(response.text))
                 # TODO: Update result to parse customMetadata when resolver returns
-                return (Entity.DataRow(self.client, {
-                    **result, 'customMetadata': []
-                }) for result in reader)
+                return (Entity.DataRow(self.client, {**result})
```

Comment:
This can just be 
```Entity.DataRow(self.client, result)```

`{**result}` is exactly the same as `result` it just makes a copy.

Reply:
Thanks, just updated